sdk/src/test/proto/replicatedentity/shoppingcart/shoppingcart_api.proto

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 Lightbend Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is the public API offered by the shopping cart replicated entity.
+
+syntax = "proto3";
+
+package com.example.replicatedentity.shoppingcart;
+
+option java_outer_classname = "ShoppingCartApi";
+
+import "akkaserverless/annotations.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+message AddLineItem {
+  string cart_id = 1 [(akkaserverless.field).entity_key = true];
+  string product_id = 2;
+  string name = 3;
+  int32 quantity = 4;
+}
+
+message RemoveLineItem {
+  string cart_id = 1 [(akkaserverless.field).entity_key = true];
+  string product_id = 2;
+}
+
+message GetShoppingCart {
+  string cart_id = 1 [(akkaserverless.field).entity_key = true];
+}
+
+message RemoveShoppingCart {
+  string cart_id = 1 [(akkaserverless.field).entity_key = true];
+}
+
+message LineItem {
+  string product_id = 1;
+  string name = 2;
+  int32 quantity = 3;
+}
+
+message Cart {
+  repeated LineItem items = 1;
+}
+
+service ShoppingCartService {
```

Comment:
I guess for continuing with codegen there will be a `option (akkaserverless.service) = { type : ... };` here as well?

Reply:
Ah, now I saw the "we need the annotations in the protocol first". NVM!

Reply:
There are some commented out annotations in the replicated entity counter sample. And some discussion in #83.